PhysObjectExtractor/src/ElectronAnalyzer.cc

```diff
@@ -192,9 +196,10 @@ ElectronAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetu
       electron_phi.push_back(el.phi());
       electron_ch.push_back(el.charge());
       electron_iso.push_back(el.ecalPFClusterIso());
-      electron_isLoose.push_back(el.electronID("cutBasedElectronID-PHYS14-PU20bx25-V2-standalone-loose"));
-      electron_isMedium.push_back(el.electronID("cutBasedElectronID-PHYS14-PU20bx25-V2-standalone-medium"));
-      electron_isTight.push_back(el.electronID("cutBasedElectronID-PHYS14-PU20bx25-V2-standalone-tight"));
+      electron_veto.push_back(el.electronID("cutBasedElectronID-Spring15-50ns-V2-standalone-veto"));//
+      electron_isLoose.push_back(el.electronID("cutBasedElectronID-Spring15-50ns-V2-standalone-loose"));
+      electron_isMedium.push_back(el.electronID("cutBasedElectronID-Spring15-50ns-V2-standalone-medium"));
+      electron_isTight.push_back(el.electronID("cutBasedElectronID-Spring15-50ns-V2-standalone-tight"));
```

Comment:
Is there a reason for taking the 50ns set of values? That stands for the time between the collision and it was 25ns for the main data taken in 2015 (https://twiki.cern.ch/twiki/bin/view/CMS/PdmV2015Analysis). 

It seems to me that the 25 ns values are listed as well.

Reply:
ok, thanks. The note you link describes the studies done for the beam behaviour before the data taking, but we know that the data were taken at the 25ns beam interval. Judging from the names of the cut-based values, two sets of them were defined before the data taking, one for 50 ns and another from 25ns, so that they would both have been available and ready to use. Unless somewhere explicitly documented, it would have been very odd that the values developed for 50ns beam interval should be used for data taken at 25ns interval,

Reply:
@katilp, ok so all the data we have for 2015 was taken at 25ns, right?   I was initially under the impression (should have checked) that we might have had a mixture.  We were looking for ways to determine whether we were looking at 25 o 50ns data with @andresc27 .  That was very instructional, but I suppose we don't have to worry about that right?

Just for the record, if one ever wants to develop a LumiAnalyzer to just check the structure of the beam and such (of course the calibration of lumi values is not correct), these link is useful:
https://twiki.cern.ch/twiki/bin/view/CMS/LumiCalc#LumiDetails

Reply:
So, @andresc27, I guess the conclusion is that we should not worry about the 50ns tags.

Reply:
ok @caredg. Just in case, I share with you the following [document](https://iopscience.iop.org/article/10.1088/1742-6596/455/1/012001/pdf) , this describes the pros and cons of using 50ns or 25 ns. Moreover this mentions the target of going for 25 ns to avoid inefficiencies.